risc0/zkvm/sdk/rust/src/build/rcc.rs

```diff
@@ -0,0 +1,614 @@
+    ///
+    /// Setting the `RCC_COMPILER` makes this method call redundant. The
+    /// presence of the environment variable supersedes this value.
+    ///
+    /// If not set via this call or the env variable above, this option is
+    /// automatically determined from the target platform or a number of
+    /// environment variables.
+    pub fn compiler<P: AsRef<Path>>(&mut self, compiler: P) -> &mut Self {
+        self.inner.compiler(compiler);
+        self
+    }
+
+    /// Configures the tool used to assemble archives.
+    ///
+    /// This option is automatically determined from the target platform or a
+    /// number of environment variables, so it's not required to call this
+    /// function.
+    pub fn archiver<P: AsRef<Path>>(&mut self, archiver: P) -> &mut Self {
+        self.inner.archiver(archiver);
+        self
+    }
+
+    /// Define whether metadata should be emitted for cargo allowing it to
+    /// automatically link the binary. Defaults to `true`.
+    ///
+    /// The emitted metadata is:
+    ///
+    ///  - `rustc-link-lib=static=`*compiled lib*
+    ///  - `rustc-link-search=native=`*target folder*
+    ///  - When target is MSVC, the ATL-MFC libs are added via
+    ///    `rustc-link-search=native=`
+    ///  - When C++ is enabled, the C++ stdlib is added via `rustc-link-lib`
+    pub fn cargo_metadata(&mut self, cargo_metadata: bool) -> &mut Self {
+        self.inner.cargo_metadata(cargo_metadata);
+        self
+    }
+
+    /// Configures whether the compiler will emit position independent code.
+    ///
+    /// This option defaults to `false` for `windows-gnu` and bare metal targets
+    /// and to `true` for all other targets.
+    pub fn pic(&mut self, pic: bool) -> &mut Self {
+        self.inner.pic(pic);
+        self
+    }
+
+    /// Configures whether the Procedure Linkage Table is used for indirect
+    /// calls into shared libraries.
+    ///
+    /// The PLT is used to provide features like lazy binding, but introduces
+    /// a small performance loss due to extra pointer indirection. Setting
+    /// `use_plt` to `false` can provide a small performance increase.
+    ///
+    /// Note that skipping the PLT requires a recent version of GCC/Clang.
+    ///
+    /// This only applies to ELF targets. It has no effect on other platforms.
+    pub fn use_plt(&mut self, use_plt: bool) -> &mut Self {
+        self.inner.use_plt(use_plt);
+        self
+    }
+
+    /// Configures whether the /MT flag or the /MD flag will be passed to msvc
+    /// build tools.
+    ///
+    /// This option defaults to `false`, and affect only msvc targets.
+    pub fn static_crt(&mut self, static_crt: bool) -> &mut Self {
+        self.inner.static_crt(static_crt);
+        self
+    }
+
+    #[doc(hidden)]
+    pub fn __set_env<A, B>(&mut self, a: A, b: B) -> &mut Self
+    where
+        A: AsRef<OsStr>,
+        B: AsRef<OsStr>,
+    {
+        self.inner.__set_env(a, b);
+        self
+    }
```

Comment:
I think this is a good public feature to expose (and needed for being able to drop the `#[serial]`.